### PR TITLE
[PD-2387] Left Spinner

### PR DESCRIPTION
**Pull request by koonsies**
To the left, to the left.

### CODE REVIEW

**Comment by tdphillips** [comment]
I'm not seeing a difference.

**Comment by koonsies** [comment]
The spinner isn't on the left of the filename now?

**Comment by tdphillips** [comment]
It was always on the left. If I go to QC, that's where it is.

**Comment by koonsies** [comment]
If you head to reports on QC, click the ellipses to the right side of an existing report and click refresh. The spinner should show up on the right side. It's easily mistaken for the spinner that appears when a new report is generated, which does appear on the left. This PR brings those two in balance.

**Comment by koonsies** [comment]
That being said, don't merge this just yet.. Checking something.

**Comment by tdphillips** [comment]
Thanks for the explanation. Just verified.

**Comment by koonsies** [comment]
After your comment, I looked back at how the "new report" spinner was populated and realized that I was creating an additional class where we didn't need one. Instead, I'm not using the same spinner case as the new report.

Less fluff code.

Works as before.

File: gulp/src/common/ui/PopMenu.jsx [hunk 1]
```diff
@@ -8,15 +8,15 @@ export default class PopMenu extends Component {
   }
 
   menuContents() {
-    const {options, isMenuActive, closeAllPopups, isMenuPending} = this.props;
+    const {options, isMenuActive, closeAllPopups} = this.props;
```
Comment (koonsies) [inline gulp/src/common/ui/PopMenu.jsx]: This and below is simply reverting back to the way the file was before this ticket.

File: static/custom.css [hunk 2]
```diff
@@ -2225,12 +2225,6 @@ h3.bottom-bordered {
   vertical-align: top;
 }
 
-.reporting.sidebar .report-loader.list-loader {
-  display: block;
-  float: right;
-  margin-top:3px;
-}
-
```
Comment (koonsies) [inline static/custom.css]: Remove styling that I added for the new class.

**Comment by aspidites** [comment]
@koonsies this PD-2387 probably needs to be started and tagged here so that it transitions correctly.

**Comment by koonsies** [comment]
Good call. Done and done!

**Comment by aspidites** [comment]
@koonsies  is this waiting on anything? I see @tdphillips  passed it, and I just verified locally that it works.

**Comment by koonsies** [comment]
Nope! Once tests pass, it should be good.

File: static/custom.css [hunk 3]
```diff
@@ -2229,12 +2229,6 @@ h3.bottom-bordered {
   vertical-align: top;
 }
 
-.reporting.sidebar .report-loader.list-loader {
-  display: block;
-  float: right;
-  margin-top:3px;
```
Comment (aspidites) [inline static/custom.css]: Were you going to re-add this, or just do it as part of the ticket dedicated to centering that icon?

**Pull request merged**